common/lib/xmodule/xmodule/course_module.py

```diff
@@ -335,6 +335,15 @@ class CourseFields(object):
         help=_("Enter the unique identifier for your course's video files provided by edX."),
         scope=Scope.settings
     )
+    facebook_url = String(
+        help=_(
+            "Enter the URL for the official course Facebook group. "
+            "If you provide a URL, the edX mobile app includes a button that students can tap to access the group."
+        ),
```

Comment:
Is it possible to have the underlying data model be more like what you expose in the API endpoint, but still keep the same UI as you have now? That would make it easier to add another service in the future.

Another idea is to make this explicitly a generic "external" URL and suggest that for example it be a facebook link - then it could be still be useful for people not using facebook integration.